Products/ZenRRD/zentestcommand.py

```diff
@@ -16,29 +16,24 @@
 
 __version__ = "$Revision$"[11:-2]
 
-import os
-import popen2
-import fcntl
-import time
-import sys
-import select
-import logging
-import signal
 from copy import copy
-log = logging.getLogger("zen.zentestcommand")
+import logging
+import sys
 
-import Globals
-from Products.ZenUtils.ZenScriptBase import ZenScriptBase
-from Products.DataCollector.SshClient import SshClient
+import Globals # noqa
+from Products.ZenUtils import Utils
+from Products.ZenUtils import ZenScriptBase
```

Comment:
This isn't typical.
Why import the module, i.e.
```
from Products.ZenUtils import ZenScriptBase
```
instead of the class, i.e.
```
from Products.ZenUtils.ZenScriptBase import ZenScriptBase
```
?

Importing the class avoids all the `ZenScriptBase.ZenScriptBase` code you have to write later.